src/chess/uciloop.cc

```diff
@@ -57,6 +57,7 @@ const std::unordered_map<std::string, std::unordered_set<std::string>>
         {{"stop"}, {}},
         {{"ponderhit"}, {}},
         {{"quit"}, {}},
+        {{"stats"}, {}},
```

Comment:
Stats is a non-standard UCI command. Instead of changing UCI interface/parser for every possible custom command, it's better to add:
```cpp
virtual bool CustomCmd(const std::string& cmd, const std::string& params) { return false; }
```
into `class UciLoop` and make it return true if `cmd` is something handled.

Reply:
I removed that command. I had a talk with borg about this some time ago, and the conclusion was that we might implement a "go nodes 0" command that does the same thing. As to not burden this rather large PR with even more details, I removed the stats command and all references.